.github/workflows/electron-survey.yml

```diff
@@ -0,0 +1,18 @@
+name: Update electron documentation survey
+on:
+  schedule:
+    - cron: "0 0 * * 1"
+
+jobs:
+  updateElectronDataPackages:
+    name: Update surveys
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/setup-node@v1
+        with:
+          node-version: '14.x'
+      - run: npm ci
+      - name: Update survey info
+        env:
+          SM_TOKEN: ${{ secrets.SM_TOKEN }}
+        run: cd .github/actions && node --unhandled-rejections=strict ./create-survey.js
```

Comment:
Need to change paths so module resolution works correctly. I didn't do the same as the other action so types are found correctly by VS Code